composer.json

```diff
@@ -67,6 +75,7 @@
     },
     "scripts": {
         "build": [
+            "./bin/laminas opcache:preload-generate",
```

Comment:
Definitely not!

You will typically only generate the preload file ONCE, and then make changes to it to customize it to your application. It should NOT be part of the build process.